libs/@local/hash-graph-types/graph_types/_schema.py

```diff
@@ -1,17 +1,22 @@
 import asyncio
 from abc import ABC, abstractmethod
 from collections.abc import Awaitable
-from typing import TYPE_CHECKING, Annotated, Any, Generic, Literal, TypeVar
+from types import EllipsisType
+from typing import TYPE_CHECKING, Annotated, Any, Generic, Literal, TypeVar, cast
 from uuid import UUID
 
 from pydantic import (
     BaseModel,
+    ConfigDict,
     Field,
-    GetCoreSchemaHandler,
     GetJsonSchemaHandler,
+    conlist,
+    create_model,
 )
 from pydantic.json_schema import JsonSchemaValue
-from pydantic_core import CoreSchema, core_schema
+from pydantic_core import CoreSchema
+
+from .base import OntologyType, OntologyTypeInfo
```

Comment:
Is `OntologyType` required for some types? Otherwise I think we can remove the import?
```suggestion
from .base import OntologyTypeInfo
```

Reply:
fixed in [8c97d8b](https://github.com/hashintel/hash/pull/2776/commits/8c97d8b9711344e7b703a28c2083e6ceeb4c66ad)